hypervisor/src/mshv/mod.rs

```diff
@@ -1327,8 +1329,21 @@ impl cpu::Vcpu for MshvVcpu {
     }
 
     #[cfg(target_arch = "aarch64")]
-    fn setup_regs(&self, _cpu_id: u8, _boot_ip: u64, _fdt_start: u64) -> cpu::Result<()> {
-        unimplemented!()
+    fn setup_regs(&self, cpu_id: u8, boot_ip: u64, fdt_start: u64) -> cpu::Result<()> {
+        let arr_reg_name_value = [(hv_register_name_HV_ARM64_REGISTER_PSTATE, 0x3c5)];
```

Comment:
What is this 0x3c5 magic value? Please document it.

Reply:
Yup good point, refactored KVM code to use the same constant.